executor/executor.go

```diff
@@ -163,6 +165,7 @@ func (e *baseExecutor) supportChunk() bool {
 	return true
 }
 
+// NextChunk fills a chunk, which is passed by reference, with multiple rows.
```

Comment:
Add the comment to the `Executor` interface is better.

Reply:
DONE